styles/components/nav-bar.scss

```diff
@@ -0,0 +1,24 @@
+.nav-bar {
+  display: inline-block;
+  width: 100%;
+  margin: auto;
+  background: $c-navbar;
+  // overflow: auto;
+
+  &__container {
+    margin: $gutter;
+  }
+
+  &__float {
+    float: left;
+    background: yellow;
+    overflow: auto;
+    }
+
+  &__list {
+    margin: $gutter;
+    padding: 0 2rem 0 0;
```

Comment:
I'm not sure I understand where the `2rem` of padding is coming from?

I think one of the issues is that the `li` within the list have the same class as the parent? This makes it so that any change made to the child will also happen to the parent.
Instead use different classes so that you can inline-block the `li` and float the parent `ul`.